packages/core/content-type-builder/admin/src/components/FormModal/attributes/types.js

```diff
@@ -5,6 +5,7 @@ import getTrad from '../../../utils/getTrad';
 import getRelationType from '../../../utils/getRelationType';
 import toRegressedEnumValue from '../../../utils/toRegressedEnumValue';
 import startsWithANumber from '../../../utils/startsWithANumber';
+import matchRegex from '../../../utils/matchRegex';
```

Comment:
I think you can remove this additional import and the file. I'll explain below why: